examples/chat.cc

```diff
@@ -6,7 +6,7 @@ using namespace handy;
 
 int main(int argc, const char* argv[]) {
     setloglevel("TRACE");
-    map<intptr_t, TcpConnPtr> users; //生命周期比连接更长，必须放在Base前
+    map<intptr_t, TcpConnPtr> users; //the life cycle is longer than the connection and must be placed before the base
```

Comment:
make comment line as an independent line

Reply:
//The life cycle of `users` is longer than the connection and must be declared before `base`.